hameedhub/banka
# #165262797 API signin Endpoint

#### What does this PR do?
Sign In the user if the credentials are correct
#### Description of Task to be completed?
- Mocha Test was carried out frist
- Static userSignin function and validation of email
#### How should this be manually tested?
After cloning the repo cd to folder /resource, you should node, intall it dependencies with **npm install** and get the RUN **npm start** to start something 
#### Any background context you want to provide?
Bcrypt affect the test result .
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165262797
